manifests/0000_09_service-ca-operator_02_crd.yaml

```diff
@@ -3,12 +3,116 @@ kind: CustomResourceDefinition
 metadata:
   name: servicecas.operator.openshift.io
 spec:
-  scope: Cluster
   group: operator.openshift.io
-  version: v1
   names:
     kind: ServiceCA
     plural: servicecas
     singular: serviceca
-  subresources:
-    status: {}
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            logLevel:
+              description: logLevel is an intent based logging for an overall component.  It
+                does not give fine grained control, but it is a simple way to manage
+                coarse grained logging choices that operators have to interpret for
+                their operands.
+              type: string
+              enum:
+                - 'Normal'
+                - 'Debug'
+                - 'Trace'
+                - 'TraceAll'
+            managementState:
+              description: managementState indicates whether and how the operator
+                should manage the component
+              type: string
+              enum:
+                - 'Force'
```

Comment:
The operator currently sets managementState to "Paused", and uses that as a no-op during the sync. For the time being we should probably allow a "Paused" value

Reply:
I've removed the enums, see note below.